object/migrator.go

```diff
@@ -26,6 +26,7 @@ func DoMigration() {
 		&Migrator_1_101_0_PR_1083{},
 		&Migrator_1_235_0_PR_1530{},
 		&Migrator_1_240_0_PR_1539{},
+		&Migrator_1_Add_Orgainzation_Prop{},
```

Comment:
Naming is wrong

Reply:
Should i name it Migrator_1_301_0_PR_1768 or something other name? I couldn't find any instructions about migrations naming.

Reply:
@aegoroff yes, that's it